### PR TITLE
Added Fedora instructions

### PR DESCRIPTION
**Pull request by nmilosev**
Added Fedora instructions for COPR. When the package gets approved, will update again.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,17 @@ sudo apt-get install yank
...
+### Fedora
+
+Versions 24/25/26:
+
+```sh
+sudo dnf copr enable nmilosev/yank
+sudo dnf install yank
+```
+
+Binary is at `/usr/bin/yank-cli`, man-pages are both `yank` and `yank-cli`.
```
Comment (mptre) [inline README.md]: Rephrase this sentence to mimic the Debian instructions:

> The binary is installed at `/usr/bin/yank-cli` due to a naming conflict,
> man-pages are available as both `yank` and `yank-cli`.

Make sure to amend or squash so we end up with a single commit.

**Comment by nmilosev** [comment]
Done, still kept the man pages information. Let me know if it is needed. :)

Cheers,
Nemanja

**Comment by mptre** [comment]
Thanks, merged! Until next time, consider:

- New sentence -> new line when editing the README ([example][1])

- Use an [imperative][2] style for commit messages

[1]: https://github.com/mptre/yank/commit/187094f5ab313583b3e76af97298f26515a735e1
[2]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

**Comment by nmilosev** [comment]
Sorry, will do!

Thanks for the merge!

**Pull request merged**